### DISCUSSION TITLE
Linux kullanıcıları için javac

### DISCUSSION THREAD

**Discussion by tatlilimon** [gusanmaz/BMB212-Algorithms-2223#15] [General]
Linux dağıtımını kullanan ve daha önce hiç java ile proje geliştirmemiş arkadaşlarım, vscode'dan java eklentisini kursa bile javac komutu tanımlanmıyor. Bu sorunu jdk'yı yükleyerek çözebilirler.

`sudo yum install java-devel `
yum yerine, kullandığınız distronun veya yüklü olan paket yöneticisinin komutunu giriniz.(apt veya dnf gibi..)

**Comment by gusanmaz** [comment]
@tatlilimon Bu ve daha onceki bilgilendirmeleriniz icin tesekkur ederim. Bu arada merak ettigim icin sormak istiyorum; bu dersi aliyor musunuz?

**Reply by tatlilimon** [reply]
Rica ederim. Evet alıyorum.

**Reply by gusanmaz** [reply]
Tamamdir. Ismini ve ogrenci numarani universite email hesabindan bana (gusanmaz [att] nku*edu* tr) yazabilir misin? Github profil bilgilerinden kim oldugunu cikaramiyorum.
